src/renderer/components/_atoms/tab/index.tsx

```
@@ -77,7 +93,10 @@
     >
       <div className='flex h-full w-full items-center gap-1 px-3 py-2 ' onClick={() => handleClickedTab()}>
         {TabIcons[languageOrDerivation]}
-        <span className='flex-grow overflow-hidden text-ellipsis whitespace-nowrap'>{fileName}</span>
+        <span
+          className='flex-grow overflow-hidden text-ellipsis whitespace-nowrap'
+          dangerouslySetInnerHTML={{ __html: handleFileName(fileName) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)